index.html

```diff
@@ -0,0 +1,127 @@
+  <title>&#x1F911 Project 3: business-site</title>
+  <!-- dont forget to add a css file and link it here! -->
+</head>
+
+<body>
+  <div class="box">
+    <div class=top-page id="top">
+
+      <!-- video or image as a header is cool :) -->
+      <div class="video">
+        <video autoplay muted loop controls class="video">
+          <source src="media/ibex.mp4" type="video/mp4">
+
+          Your browser does not support the video tag.
+        </video>
+      </div>
+      <div class="content">
+        <h1 class="content">Photagraphy Class</h1>
+        <p class="content">We love photagrphy and we love people. please join us to share the love.
+          <br>The First Permanent Images
+          Photography, as we know it today, began in the late 1830s in France. Joseph Nicéphore Niépce used a portable
+          camera obscura to expose a pewter plate coated with bitumen to light. This is the first recorded image that
+          did not fade quickly.
+        </p>
+      </div>
+    </div>
+
+    <!-- ***Accordion, FAQ*** -->
+    <section class="accordion" id="faq">
+      <!-- Accordion -->
+      <h1>FAQ</h1>
+      <div class="container">
+        <div class="question" id="section1">
+
+          <p>Do I need to own a camera?</p>
+
+        </div>
+        <div class="answer">
+          <p>Since you start out as a beginer, the camera in your smartphone is fine. If you need advice
+            on which camera is good to buy you can always ask any of our teachers.</p>
+
+        </div>
+      </div>
+      <div class="container">
+
+        <div class="question" id="section2">
+
+          <p>How much does it cost?</p>
+
+
+        </div>
+        <div class="answer">
+          <p>One semester is 50 dollars and two is 75.</p>
+        </div>
+      </div>
+      <div class="container">
+        <div class="question" id="section3">
+
+          <p>What is the schedual?</p>
+
+
+        </div>
+        <div class="answer">
+          <p>The class is one day in the afternoon for 2h.</p>
+        </div>
+
+      </div>
+    </section>
+    <!-- Signup form -->
+    <section class="input" id="input">
+      <form action="https://httpbin.org/anything" method="POST">
+        <div class="container">
+          <h1>Register</h1>
+          <p>Please fill in this form for more information regarding the class and how to participate.</p>
+          <hr>
+          <label for="name">Name</label>
+          <input type="text" placeholder="First Name Last Name" name="name" required>
+
+          <label for="email">Email</label>
+          <input type="text" placeholder="Enter Email" name="email" required>
+
+          <label for="phone">Phone Number</label>
+          <input type="text" placeholder="+46 10 101 0101" name="phone" required>
+          <label for="name">Course Level</label>
+          <select name="course" id="course">
+            <option value="b">Beginer</option>
```

Comment:
A good practice is to use shortcut only when you understand what it means. "b" is kind of hard to understand by its own what it is
```suggestion
            <option value="beginner">Beginner</option>
```